src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1256,6 +1239,31 @@ static void uct_dc_mlx5_iface_handle_failure(uct_ib_iface_t *ib_iface,
     }
 }
 
+
+static int
+uct_dc_mlx5_iface_is_reachable_v2(const uct_iface_h tl_iface,
```

Comment:
why needed to move this function from lines 916 to here?

Reply:
moved to 916